apps/liteprotocoltester/Dockerfile.liteprotocoltester.compile

```diff
@@ -0,0 +1,61 @@
+# BUILD NIM APP ----------------------------------------------------------------
+  FROM rust:1.77.1-alpine3.18  AS nim-build
+
+  ARG NIMFLAGS
+  ARG MAKE_TARGET=liteprotocoltester
```

Comment:
waku.nims takes target name and use it as the main source name for build... so for this we would need to name project main source like this...WDYT?

Reply:
> waku.nims takes target name and use it as the main source name for build... so for this we would need to name project main source like this...WDYT?

Yes, I think it would get a bit more readable in hyphen-separated mode